server-spi-private/src/main/java/org/keycloak/forms/login/LoginFormsPages.java

```diff
@@ -27,6 +27,7 @@
     OAUTH_GRANT, LOGIN_RESET_PASSWORD, LOGIN_UPDATE_PASSWORD, LOGIN_SELECT_AUTHENTICATOR, REGISTER, REGISTER_USER_PROFILE, INFO, ERROR, ERROR_WEBAUTHN, LOGIN_UPDATE_PROFILE,
     LOGIN_PAGE_EXPIRED, CODE, X509_CONFIRM, SAML_POST_FORM,
     LOGIN_OAUTH2_DEVICE_VERIFY_USER_CODE, UPDATE_USER_PROFILE, IDP_REVIEW_USER_PROFILE,
-    FRONTCHANNEL_LOGOUT;
+    FRONTCHANNEL_LOGOUT,
+    LOGIN_RECOVERY_AUTHN_CODES_INPUT, LOGIN_RECOVERY_AUTHN_CODES_CONFIG;
```

Comment:
Should rather just use the update credentials event than to introduce a dedicate even for recovery codes

Reply:
@stianst I think this Enum drives the Freemaker templates and not the events. I don't think we created custom event for recovery codes